DEV/org.openl.rules/src/org/openl/rules/lang/xls/TableSyntaxNodeUtils.java

```diff
@@ -95,28 +95,32 @@ public static String str2name(String methodHeader, XlsNodeTypes tableType) {
         if (StringUtils.isBlank(resultName)) {
             resultName = "NO NAME";
 
-        } else if (tableType.equals(XlsNodeTypes.XLS_DATATYPE)) {
-            String[] tokens = StringUtils.split(resultName.replaceAll(ROUND_BRACKETS_WITH_ANY_TEXT, ""));
-            // ensure that the appropriate index exists
-            //

```

Comment:
It seems that the code is over-complicated for just extracting of the table name. Why is it needed to convert `methodHeader` to bunch of object if it is enough to determine the first-last index of the token and do `substring` ?